npm/src/registry.ts

```diff
@@ -29,6 +29,12 @@ export interface Rpc {
   images: Image[];
 }
 
+export interface Frontend {
+  name: string;
+  url: string;
+  images: Image[];
+}
```

Comment:
thought: I wonder if we can merge Rpc and Frontend types to a kind of Identity type?

Reply:
Came up (with the help of AI) with `Gateway` as the word that can apply both to Rpc and Frontend 😅